### DISCUSSION TITLE
Upcoming Build 1.3.0 and Hiatus

### DISCUSSION THREAD

**Discussion by hargata** [hargata/lubelog#451] [Announcements]
Good day, the following features and enhancements have been planned for 1.3.0:

- Confirmation Dialog before closing modal window if content has changed.
- Increment last reported odometer reading.
- Global search
- Ability to edit existing planner template instead of having to delete/re-create

The release date for 1.3.0 is still TBD but ideally before May 1st, which leads us to the next topic at hand.

We're taking a hiatus from May to November, we're probably still going to pop in once a while to answer some questions or even make minor commits to the code but it is unlikely there will be a release until well past November 1st. There are multiple reasons for this hiatus, but mainly because we have other, more pressing obligations in the Summer. 

We're announcing this so that our users don't think we've abandoned the project.

If you have a feature request that needs **immediate attention** anywhere between May and November, it is unlikely that we will get to it without some serious incentives. You may contact us at hargatasoftworks@gmail.com to make such arrangements but do note that any work that is performed during this time frame has a going-rate of $200/hour and a minimum of 40 hours. Yeap, unless you're paying us a minimum of $8000, your feature request can wait until we come back from our hiatus.

Thank you.

**Comment by Eschguy** [comment]
Enjoy the time off!

**Comment by varunpan** [comment]
Sounds awesome! Enjoy your time off! Thank you for adding the ability to edit/remove the planner templates in this next release before your time off :D

**Comment by gt2416** [comment]
One of the best open sources software I've seen develop so fast ! Absolutely love it! 
Can't wait for the new versions. 
Enjoy your time off !

**Comment by contrerasp** [comment]
Has anyone paid up yet? lol hope vacation is going well!!!

**Reply by hargata** [reply]
We had one feature that was funded in mid-August(adding extra fields to the CSV Export) and we've used the opportunity to work on other smaller tickets during that period that are included in the release of 1.3.6
